plugins/woocommerce-admin/client/layout/controller.js

```diff
@@ -187,7 +190,21 @@ export const getPages = () => {
 		} );
 
 		pages.push( {
-			container: EditProductPage,
+			container: AddProductPage,
+			path: '/add-product-old',
```

Comment:
If I access the old editor here, it's missing both heading bars:

![image](https://user-images.githubusercontent.com/444632/222826552-97e166a0-3c3f-457d-9e23-d68bfbe2b931.png)

Reply:
Good catch! It's done now.